src/main/scala/com/johnsnowlabs/reader/EmailReader.scala

```diff
@@ -27,6 +27,44 @@ import java.util.Properties
 import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
 
+/** This class is used to read and parse email content.
+  *
+  * @param addAttachmentContent
+  *   Whether to extract and include the textual content of plain-text attachments in the output.
+  *   By default, this is set to false.
+  * @param storeContent
+  *   Whether to include the raw file content in the output DataFrame as a separate 'content'
+  *   column, alongside the structured output. By default, this is set to false.
+  *
+  * ==Example==
+  * {{{
+  * val emailsPath = "./email-files/test-several-attachments.eml"
+  * val emailReader = new EmailReader()
+  * val emailDf = emailReader.read(emailsPath)
+  * }}}
+  *
+  * {{{
+  * emailDf.show()
+  * +--------------------+--------------------+
+  * |                path|               email|
+  * +--------------------+--------------------+
+  * |file:/content/ema...|[{Title, Test Sev...|
+  * +--------------------+--------------------+
+  *
+  * emailDf.printSchema()
+  * root
+  *  |-- path: string (nullable = true)
+  *  |-- email: array (nullable = true)
+  *  |    |-- element: struct (containsNull = true)
+  *  |    |    |-- elementType: string (nullable = true)
+  *  |    |    |-- content: string (nullable = true)
+  *  |    |    |-- metadata: map (nullable = true)
+  *  |    |    |    |-- key: string
+  *  |    |    |    |-- value: string (valueContainsNull = true)
+  * }}}
+  *
+  * For more examples please refer - examples/python/reader/SparkNLP_Email_Reader_Demo.ipynb
```

Comment:
You can actually link to these notebooks directly. For Example:

```
[[https://github.com/JohnSnowLabs/spark-nlp/blob/master/examples/python/training/english/dl-ner/ner_bert.ipynb Examples]]
```

Just pretend the file is already in master with the link.

You can also adjust the other links.

Reply:
Done